pkg/apis/harvesterhci.io/v1beta1/image.go

```diff
@@ -39,9 +40,12 @@ type VirtualMachineImageSpec struct {
 	DisplayName string `json:"displayName"`
 
 	// +kubebuilder:validation:Required
-	// +kubebuilder:validation:Enum=download;upload
+	// +kubebuilder:validation:Enum=download;upload;export-from-volume
 	SourceType string `json:"sourceType"`
 
+	// +optional
+	VolumeName string `json:"volumeName"`
```

Comment:
PersistentVolumeClaimName might be better. I think the namespace is also needed

Reply:
@gitlawr updated